ebean-core/src/main/java/io/ebeaninternal/server/core/OrmQueryRequest.java

```diff
@@ -579,6 +579,15 @@ private ElPropertyValue mapProperty() {
     return property;
   }
 
+  @Override
+  public Set<T> beanCacheHitsAsSet() {
+    OrderBy<T> orderBy = query.getOrderBy();
+    if (orderBy != null && !orderBy.isEmpty()) {
+      beanDescriptor.sort(cacheBeans, orderBy.toStringFormat());
```

Comment:
should we support ordering on sets? Maybe return a normal hashSet, if no orderBy is set

Reply:
Happy to just use LinkedHashSet here.  No need to add HashSet use here imo.